Mathlib/Analysis/Normed/Algebra/Exponential.lean

```diff
@@ -125,6 +131,10 @@ theorem expSeries_sum_eq (x : 𝔸) : (expSeries 𝕂 𝔸).sum x = ∑' n : ℕ
 theorem exp_eq_tsum : exp 𝕂 = fun x : 𝔸 => ∑' n : ℕ, (n !⁻¹ : 𝕂) • x ^ n :=
   funext expSeries_sum_eq
 
+/-- The exponential sum as an `ofScalarsSum`. -/
+theorem exp_ofScalars : exp 𝕂 = ofScalarsSum (E := 𝔸) fun n ↦ (n !⁻¹ : 𝕂) := by
```

Comment:
Not your fault but `n !⁻¹` is ugly 😞

Reply:
yep I agree :(, it took me a couple of minutes to parse the notation the first time I saw it